include/fiction/algorithms/simulation/sidb/time_to_solution.hpp

```diff
@@ -105,14 +105,14 @@ void time_to_solution(Lyt& lyt, const quicksim_params& quicksim_params, const ti
     sidb_simulation_result<Lyt> simulation_result{};
     if (tts_params.engine == exhaustive_sidb_simulation_engine::QUICKEXACT)
     {
+        const quickexact_params<Lyt> params{quicksim_params.phys_params};
         st.algorithm      = "QuickExact";
-        simulation_result = exhaustive_ground_state_simulation(lyt, quicksim_params.phys_params);
+        simulation_result = quickexact(lyt, params);
     }
     else
     {
-        const quickexact_params<Lyt> params{quicksim_params.phys_params};
-        st.algorithm      = "QuickExact";
-        simulation_result = quickexact(lyt, params);
+        st.algorithm      = "Exhaustive Ground State Simulation";
```

Comment:
I believe we use `ExGS` as the algorithm name here. @Drewniok can you confirm this?

Reply:
@marcelwa You are right!
@wlambooy Please use ``ExGS``. Thanks